lib/array_enum.rb

```diff
@@ -9,21 +9,26 @@ module ArrayEnum
   def array_enum(definitions)
     definitions.each do |attr_name, mapping|
       attr_symbol = attr_name.to_sym
+      mapping_hash = ActiveSupport::HashWithIndifferentAccess.new(mapping)
```

Comment:
could you add `require "active_support/hash_with_indifferent_access"` on top of file?